scripts/ci/install-dev-deps

```diff
@@ -25,4 +26,7 @@ def run(command):
 
 if __name__ == "__main__":
     with cd(REPO_ROOT):
-        run("pip install -r requirements-dev-lock.txt")
+        if sys.version_info[:2] == (3, 12):
+            run("pip install setuptools")
+
+    run("pip install -r requirements-dev-lock.txt")
```

Comment:
Why does this line need to be run outside the context manager now?

Reply:
Should already fixed in [3b493d0](https://github.com/boto/botocore/pull/2967/commits/3b493d03cb2e0a78fc008790c22dc02fe1841771).